### PR TITLE
Github action deploy to kubeflow

### PR DESCRIPTION
**Pull request by NikeNano**
The goal with this example is to show how GitHub actions can be used for CI/CD/MLops together with kubeflow. It is build upon this [action](https://github.com/marketplace/actions/kubeflow-compile-deploy-and-run). Super happy for feedback on the action is self as well! 

With this example I hope so show how the action can be used and improve CI/CD. 

This likes to issue: https://github.com/kubeflow/examples/issues/711

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/730)

### CODE REVIEW

**Comment by NikeNano** [comment]
I am not done yet but wanted to do a PR before the doc-sprint was over. Will continue the work and hopefully have it done tomorrow :) Will let you know when it is done!

**Comment by NikeNano** [comment]
@JohanWork did you want to check it our already now?

File: pipelines/github_action/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+```
+ 
+ Github workflows can be given names, and in the example the name is set to: "Compile, Deploy and Run on Kubeflow". This name will then be the name of the action when it runs on Github. 
+
+ The ON arguments is replate to which actions on github should this workflow be triggered on. For more info see [here](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#on)
+
+"Runs on" defines which type of machine should the workflow be executed on, in this case it dont matter since we will use a action(NikeNano/kubeflow-github-action@master) which are containerize. 
+
+A Github workflow is splitted to steps. In this example the first step will check out the code. This is needed in order to be abe to access the source code to the repo which the workflow is applied to. The firts action used is named "uses: actions/checkout@master", master here referst to the master branch of [the repository](https://github.com/actions/checkout) where this action is open sourced.
```
Comment (jlewi) [inline pipelines/github_action/README.md]: type able to access

**Comment by JohanWork** [comment]
@googlebot I fixed it.

**Comment by JohanWork** [comment]
@googlebot I consent.

**Comment by jlewi** [comment]
please rebase your changes off the latest master to pick up a fix to the failing test.

**Comment by NikeNano** [comment]
@jlewi I have rebased and fixed listing. @sarahmaddox, @amygdala  I am done with what I wanted to do, so ready for review :)

**Comment by jlewi** [comment]
/lgtm
/approve
/hold 
In case anyone else wants to review this

**Comment by sarahmaddox** [comment]
@jlewi It looks like no-one else wants to review. Can we remove the hold now?

**Comment by NikeNano** [comment]
Cloud we marge this @jlewi ?

**Comment by jinchihe** [comment]
/hold cancel

Thanks @NikeNano

**Comment by NikeNano** [comment]
Great @jinchihe :) Happy to contribute

**Pull request merged**